### DISCUSSION TITLE
Getting NaN after running AI. Is my training data wrong?

### DISCUSSION THREAD

**Discussion by dominic-black** [BrainJS/brain.js#829] [Q&A]
Hello.
My training data looks as follows:
```
    input: [
      "Sett, Galio, Lee Sin, Miss Fortune, Pyke",
      "Pantheon, Aphelios, Kai'Sa, Fiddlesticks, Jhin",
    ],
    output: [1],
```
and I'm passing net.run(testData) where testData is
`[ "Gwen, Graves, Gragas, Gnar, Jayce", "Darius, Jax, Kayn, Lux, Neeko" ]`.

Can anyone see why I would be getting NaN as a result?

Thank you.

**Comment by robertleeplummerjr** [comment]
That only works with a LSTM.
